stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -542,6 +542,11 @@ public func expectCrashLater() {
   _seenExpectCrash = true
 }
 
+public func expectCrash(executing: () -> Void) {
+  expectCrashLater()
+  executing()
```

Comment:
I thought of this too late, but would it make sense to put `expectUnreachable()` after the call to `executing`?

Reply:
`expectUnreachable` might be sufficient there. Then we can make `expectCrash` return `Never`, and we'll get unreachable warnings for anything that's written after the first `expectCrash`.